backend/routes/db.py

```diff
@@ -257,9 +259,8 @@ async def get_concept_ids_by_codeset_id_post(request: Request, codeset_ids: Unio
 @return_err_with_trace
 async def get_concept_ids_by_codeset_id(
     request: Request, codeset_ids: Union[List[str], None] = Query(...)
-) -> Dict[str, str]:
+) -> Dict[int, List[int]]:
```

Comment:
## The bug fix
The typedef was wrong for this function. But IDK why it's erroring now whereas it was not before. It doesn't seem like our `requirements.txt` has changed very much, but probably one or more packages got upgraded in the past few months which was causing it to now do these validations.